templates/cisco_s300_show_version.textfsm

```diff
@@ -0,0 +1,21 @@
+Value SW_VERSION ([1-9]\.[0-9]\.[0-9].[0-9]*)
+Value BOOT_VERSION ([1-9]\.[0-9]\.[0-9].[0-9]*)
+Value HW_VERSION (V0[1-9])
+
+Start
+  ^SW\sversion\s+${SW_VERSION}\s.* -> FWVER1
```

Comment:
Do you minding changing all the `\s` to `\s+` just in case Cisco changes whitespace between versions (not unheard of) and since we're expecting at least one space is the reason for the `+`

Reply:
Although I don't see why Cisco would increase the whitespace between "SW version", "Boot version" and "HW version" in the v1.x.y.z output, no harm covering the bases I guess :)